### ISSUE TITLE
Support locally encrypted file storage

### ISSUE THREAD

**Issue by tyvsmith** [uber/simple-store#94]
SImple-store should support locally encrypted files to store contents like authorization tokens.

Can take inspiration from https://developer.android.com/reference/androidx/security/crypto/EncryptedSharedPreferences

**Comment by kurtisnelson** [comment]
It doesn't care what the payload is as long as it is a byte[], so no need for key management to be built in.

**Comment by tyvsmith** [comment]
Yeah, using byte for storage makes sense. Tentatively, I'd expect a wrapper/util on top to facilitate easy to use APIs for consumers.
